### PR TITLE
Ray logs plugin

### PR DESCRIPTION
**Pull request by PabloCarmona**

### CODE REVIEW

**Comment by starpit** [comment]
thanks! can you attach some screenshots of the features?

File: plugins/plugin-client-default/notebooks/dashboard.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+```bash
+---
+execute: now
+---
+ray job logs raysubmit_mP8gXdLKVTkdEwfX
+```
+
+---
+
+```bash
```
Comment (atinsood) [inline plugins/plugin-client-default/notebooks/dashboard.md:26]: @PabloCarmona I think we can remove this section for now till we have this implemented.

File: plugins/plugin-client-default/notebooks/dashboard.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+  1: default
+  2: default
+  3: default
+---
+
+```bash
+---
+execute: now
+---
+ray job info raysubmit_mP8gXdLKVTkdEwfX
```
Comment (atinsood) [inline plugins/plugin-client-default/notebooks/dashboard.md:12]: @starpit do you think we should hardcode some random ids for this commit or you have a variable in mind that we can use

File: plugins/plugin-client-default/src/components/CurrentJob.tsx [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+    this.state = {
+      currentJob: 'Loading...',
+      viewLevel: 'normal',
+      jobs: {},
+      currentTab: undefined
+    }
+  }
+
+  retrieveRunningJobs = async () => {
+    const response = await fetch(`http://127.0.0.1:8265/api/jobs/`)
```
Comment (atinsood) [inline plugins/plugin-client-default/src/components/CurrentJob.tsx:43]: @starpit @PabloCarmona can we use a variable to control where the jobs server is running. 

may be separately if the job server is not running, we might want to capture that error and render that somehow.

**Comment by PabloCarmona** [comment]
Here they go some screenshots @starpit:
* StatusBar with current Ray job:

Captura de Pantalla 2022-06-13 a las 12 00 38

* This launches when you click in the StatusBar Ray Job ID:
Captura de Pantalla 2022-06-13 a las 12 00 50

* This is the logs view:
Captura de Pantalla 2022-06-13 a las 12 01 11

* This is the list of the jobs lauchend, still a work in progress to be presented as a table:
Captura de Pantalla 2022-06-13 a las 12 01 36

File: plugins/plugin-client-default/notebooks/dashboard.md (hunk 2 above)
Comment (PabloCarmona) [reply plugins/plugin-client-default/notebooks/dashboard.md:12]: This is a thing that I need to figure it out with @starpit in terms on how we pass dynamic variables to the markdown files if is possible

File: plugins/plugin-client-default/src/components/CurrentJob.tsx (hunk 3 above)
Comment (PabloCarmona) [reply plugins/plugin-client-default/src/components/CurrentJob.tsx:43]: Here we can have some config file is not created yet to use different BASE_URL to connect server in development or production

File: plugins/plugin-client-default/notebooks/dashboard.md (hunk 2 above)
Comment (starpit) [reply plugins/plugin-client-default/notebooks/dashboard.md:12]: i think we will be working in one of two modes:

1) the ui is launched in the context of a given job. in which case there is nothing to hard-code, rather we would access the current job id. e.g. passed as an env var. or simply as a command line argument.
2) the user wants to switch to some other running or previously run job. 

perhaps `ray job logs` without any arguments should use the current job.

File: plugins/plugin-client-default/src/components/CurrentJob.tsx (hunk 3 above)
Comment (starpit) [reply plugins/plugin-client-default/src/components/CurrentJob.tsx:43]: same as with the job id

**Pull request closed without merging**